### ISSUE TITLE
One Stop Shop Homeless SMS Resource

### ISSUE THREAD

**Issue by etanb** [sfbrigade/project-ideas#1]
Background: 

I'm working with LavaMae on a project and this is where this idea stems from.

One sentence description:

Simple SMS directory for basic needs like internet, showers (Lavamae + other shelters), food etc. 

Link (more details/brain dump/alpha):

N/A

Project Needs (dev/design/resources):

This will only be SMS based so no design needs. The real cost here is finding some API equivalent to Twilio that's free. Twilio gets expensive really quick. Also, help building the Admin side so partnering organizations can input their information.

Status (in progress, pie-in-the-sky)

Pie all up in the sky :)

**Comment by ondrae** [comment]
Hi @etanb 

This is a great idea. Code for San Francisco has already built some of the first pieces of this with [Local Free Web](http://localfreeweb.org/). Adding more than just free web to it could be useful. Either add more data in or make a complimentary service like Local Free Baths.

Code for San Francisco gets access to a pile of donated Twilio credits. Once this project is in it a place that it needs them, let me or @jrkao know.

**Comment by etanb** [comment]
Thank you for the response @ondrae !

Is it possible to get some credits for testing? Also, any help for hosting costs for a project like this or recommendations on keeping it as cheap as possible.

I think this should be a standalone project but their code is definitely useful.

Thank you all so much!

**Comment by gitsteph** [comment]
@etanb !  Are you still working on this?  Love to help if you need some devs!  I already have some additional ideas around how to implement/experience working with Twilio.  :)

**Comment by etanb** [comment]
@gitsteph Hey! I actually had to stop due to time overload...but I'd LOVE to start this up again. Are you interested?

**Comment by gitsteph** [comment]
@etanb : Yes!  Please send me an email (byronic.string@gmail.com) so we can chat further (online, but off-forum). I actually started getting into coding shortly after being inspired by the LavaMae poop map in 2014 (http://mashable.com/2014/11/21/public-poop-san-francisco/#SbIDiYRUjEqb) -- I'm excited to hear another way to help the cause!

**Comment by 808sAndBR** [comment]
I am not sure if this exactly fits in with your plan for this, but I know people that built something similar for finding stores/restaurants that accept SNAP (food stamps) in San Francisco. It has a map interface that you can use via the web but you can also text it your address and it will send back the closest x (I think 3 or 5) locations.

They did it at a hackathon so it might be a bit messy but I remember that cleaning the data took a lot of their time so it might be useful at least for that part?

Web interface: http://shrouded-everglades-4199.herokuapp.com/
github repo: https://github.com/ajayvaria/ebthelp
